clang-format-changed-files.sh

```diff
@@ -2,9 +2,10 @@
 
 # This script runs clang_format on all .h, .m, and .mm files that have changes from develop branch.
 # Before running this, ensure that your local develop branch is up to date.
-git fetch
+upstream=$(git remote -v | grep "Microsoft" | awk '{print $1}' | uniq)
```

Comment:
nit: ":Microsoft/" to avoid matching users that would use Microsoft somewhere in their account or org.